### ISSUE TITLE
Unraid template

### ISSUE THREAD

**Issue by Brramble** [henrygd/beszel#9]
Hey, please could this be added to the unraid community store? Or, with your permission could I request it to be added by a maintainer?

**Comment by JamsRepos** [comment]
Just added one to the store, should be available in a couple hours.

**Comment by henrygd** [comment]
I don't use Unraid so I'm 100% fine with it being managed by others as long as there's no security concerns associated with that. And it's able to be updated in the future.

**Comment by JamsRepos** [comment]
Yeah absolutely, it'll auto-update to any latest release. Any new/changes to environment/labels then i'll of course need to update but open to PR's if I miss it.

**Comment by Heavensong89** [comment]
the /var/run/docker.sock should be set to Read-Only ideally, just seen the template drop in the store, saved me having to do my own container setup! 😄

**Comment by JamsRepos** [comment]
> the /var/run/docker.sock should be set to Read-Only ideally, just seen the template drop in the store, saved me having to do my own container setup! 😄

Good catch, i'll update it now. It'll roll out within 2 hours.

**Comment by doomedramen** [comment]
Cheers @jamsrepos

**Comment by henrygd** [comment]
Sounds like this has been worked out. I'm going to close the issue and convert it to a discussion in case anyone wants to talk about it in the future.

If you need anything more from me, let me know.
